<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formLogin</class>
 <widget class="QDialog" name="formLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>578</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>561</width>
    <height>578</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>561</width>
    <height>578</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>GEM - Tela de Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../images/logo_hepp.png</normaloff>../../images/logo_hepp.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: rgb(0, 74, 112);
    font-family: 'Segoe UI';
}

QLineEdit {
    border: 1px solid #aaa;
    border-radius: 6px;
    padding: 8px;
    background-color: #fff;
    font-size: 11px;
    font-weight: bold;
}

QLineEdit:focus {
    border: 2px solid #2D89EF;
    background-color: #f8f9ff;
}

QPushButton {
    background-color: #2D89EF;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 10px 20px;
    font-weight: bold;
    font-size: 11px;
}

QPushButton:hover {
    background-color: #1A6FD8;
}

QPushButton:pressed {
    background-color: #0F4B9B;
}

QPushButton#btnFechar {
    background-color: #6c757d;
}

QPushButton#btnFechar:hover {
    background-color: #5a6268;
}

QPushButton#btnFechar:pressed {
    background-color: #4e555b;
}
</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="frameLogin">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>361</width>
     <height>221</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frameLogin {
    background-color: rgba(0, 0, 0, 0.2);
    border-radius: 10px;
    border: 1px solid rgba(255, 255, 255, 0.1);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <widget class="QLineEdit" name="txtUser">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>241</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>üë§ Usu√°rio</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPassword">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>241</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>üîí Senha</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEntrar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>üöÄ Entrar</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFechar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>‚ùå Fechar</string>
    </property>
    <property name="shortcut">
     <string>Esc</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblLogo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: white;
    background-color: transparent;
}</string>
   </property>
   <property name="text">
    <string>GEM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblSubtitulo">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>77</y>
     <width>431</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: rgba(255, 255, 255, 0.8);
    background-color: transparent;
}</string>
   </property>
   <property name="text">
    <string>Gerenciamento de Escalas M√©dicas</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCadeado">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>181</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../images/cadeado_login.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblEsqueciMinhaSenha">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: rgba(255, 255, 255, 0.9);
    background-color: transparent;
}

QLabel:hover {
    color: #FFD700;
    background-color: rgba(255, 215, 0, 0.1);
    border-radius: 5px;
    padding: 2px;
}</string>
   </property>
   <property name="text">
    <string>üîë Esqueci minha senha</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblVersao">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: rgba(255, 255, 255, 0.6);
    background-color: transparent;
}</string>
   </property>
   <property name="text">
    <string>v1.0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>lblCadeado</zorder>
  <zorder>frameLogin</zorder>
  <zorder>lblLogo</zorder>
  <zorder>lblSubtitulo</zorder>
  <zorder>lblEsqueciMinhaSenha</zorder>
  <zorder>lblVersao</zorder>
 </widget>
 <tabstops>
  <tabstop>txtUser</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>btnEntrar</tabstop>
  <tabstop>btnFechar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
